src/components/Restoration/ExtendTimeLimit.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <v-card flat id="extend-time-limit" class="pt-2 mr-8">
+
+    <!-- Extension Time Section -->
+    <section
+      id="extension-time-section"
+      class="section-container"
+      :class="{ 'invalid-section': invalidSectionExpiry }"
+    >
+      <v-row no-gutters>
+        <v-col cols="12" sm="3">
+          <strong :class="{ 'error-text': invalidSectionExpiry }">Extension Time</strong>
+        </v-col>
+        <v-col cols="12" sm="9">
+          <!-- Limited Restoration Radio Panel -->
+          <LimitedRestorationPanel
+            :currentDate="getCurrentDate"
+            :expiryDate="expiry"
+            :key="expiry"
```

Comment:
Is this because the component doesn't update itself when the `expiryDate` prop changes? Normally we'd handle that with a watcher in the component, but that's a bit messy. This, however, feels like a cheat :)

I don't know how long Jonathan is away for but I'd like his opinion on this.

My opinion is this OK until we have a better way.

Reply:
Yes, exactly! This way whenever the prop data changes (getters, setters etc), the component will update.

CC: @jonathan-longe

Reply:
This appears to be an issue for all instances of the `<LimitedRestorationPanel>`.  As such, I'd suggest we should create a ticket to resolve it rather than handling it in the parent.  Please push back if you disagree.